src/dataspec/base.py

```diff
@@ -505,19 +586,20 @@ def validate(self, d) -> Iterator[ErrorDetails]:  # pylint: disable=arguments-di
             if k in d:
                 yield from _enrich_errors(vspec.validate(d[k]), self.tag, k)
 
-    def _default_conform(self, d):  # pylint: disable=arguments-differ
-        conformed_d = {}
-        for k, spec in self._reqkeyspecs.items():
-            conformed_d[k] = spec.conform(d[k])
-
-        for k, spec in self._optkeyspecs.items():
-            if k in d:
-                conformed_d[k] = spec.conform(d[k])
 
-        return conformed_d
+class ObjectSpec(DictSpec):
+    @classmethod
+    def from_val(
```

Comment:
Should this come from a mixin? `NoDefaultConformerMixin`

Reply:
Hrm. It's an interesting idea, though I'm not sure it's necessary at the moment (given this is the only case like this).

Actually your comment has me wondering why I need this at all. Can't we just pass the object through without a conformer? If the caller wants to provide a conformer, that's their prerogative, but I see no reason to throw an error by default here.

Reply:
I removed the default conformer nonsense. Now, we just completely override the `DictSpec` conformer with either `None` or the user-provided conformer.